docs/rules/no-sequences.md

```diff
@@ -66,7 +66,7 @@ while ((val = foo(), val < 42));
 ## When Not To Use It
 
 Disable this rule if sequence expressions with the comma operator are acceptable.
-Another case where you might want to reports all usages of the comma operator, even if they are wrapped in parenthesis or in a for loop. You can achieve this using rule `no-restricted-syntax`:
+Another case is where you might want to reports all usages of the comma operator, even if they are wrapped in parenthesis or in a for loop. You can achieve this using rule `no-restricted-syntax`:
```

Comment:
Typo: parenthesis (should be parentheses; "parenthesis" is the singular noun, while "parentheses" is the plural)